.github/workflows/ci.yml

```diff
@@ -120,38 +120,32 @@ jobs:
         working-directory: integration-tests
         run: cargo test --release --locked --features "${{ matrix.features }}"
 
-  api-tests:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: "./.github/actions/init"
-
-      - name: Run tests
-        working-directory: pop-api
-        run: cargo test --release --locked --all-features
-
-  api-integration-tests-devnet:
+  pop-api-integration-tests:
+    needs: lint
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        features:
```

Comment:
Nice clean up. Might be worth adding a comment as to why these have to be run separately. Will just help anyone needing to improve in the future by being explicit.

Reply:
https://github.com/r0gue-io/pop-node/pull/434/commits/d44ac5618e3b98190c0382fc5efceef7f705c53e